psysh.el

```diff
@@ -188,6 +203,39 @@ See `psysh-mode-output-syntax-table'."
     (psysh)))
 
 
+;; History
+(defun psysh--config-dir-path ()
+  "Return path to PsySH config dir."
+  ;; TODO: maybe next version Emacs bundles xdg.el?
+  (if (eq system-type 'windows-nt)
+      (f-join (s-replace-all '(("\\" . "/"))
+                             (or (getenv "APPDATA")
+                                 (f-join (getenv "HOME") "AppData"))) "PsySH")
+    (f-join (or (getenv "XDG_CONFIG_HOME")
+                (f-join (getenv "HOME") ".config"))
+            "psysh")))
+
+(defun psysh--load-history (path n)
+  "Load input histories by PATH and return N elements."
```

Comment:
History as a "list of past choices" is always singular, `histories` means "multiple stories", e.g. "Rome, Greece and Ancient China have interesting histories".